backend/LexBoxApi/Services/HgService.cs

```diff
@@ -326,7 +329,9 @@ public async Task<string> ExecuteHgRecover(string code, CancellationToken token)
 
     private async Task<string> ExecuteHgCommandServerCommand(string code, string command, CancellationToken token)
     {
-        var httpClient = _hgClient.Value;
+        // var httpClient = _hgClient.Value;
+        // TEMPORARY CHANGE, revert this once we finish running the "update all lex entry counts" migration
+        var httpClient = _clientFactory.CreateClient("HgWeb");
```

Comment:
rather than this hack we should just not use `httpClient` base address and just prefix the `GetAsync` url call below to use the command server.